src/templates/_includes/os_x_banner.html

```diff
@@ -0,0 +1,8 @@
+<mkt-banner id="mac-install-notice" success>
```

Comment:
i think i prefer classes due to the potential CSS specificity issues using IDs may introduce (even though we're not styling it here)